src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccountController.cs

```diff
@@ -68,11 +69,12 @@ public class AccountController : Controller {
             }
 
             var membershipSettings = _membershipService.GetSettings();
+
             if (user != null &&
                 membershipSettings.EnableCustomPasswordPolicy &&
                 membershipSettings.EnablePasswordExpiration &&
                 _membershipService.PasswordIsExpired(user, membershipSettings.PasswordExpirationTimeInDays)) {
-                return RedirectToAction("ChangeExpiredPassword", new { username = user.UserName });
+                return RedirectToAction("ChangeExpiredPassword", "Account", new { Area = "Orchard.Users", username = user.UserName });
```

Comment:
In the same PR you change this to the Users module, but you also create routes for the OpenId module, on the same Controller/Action names, is that on purpose?